recipes/src/main/resources/quarkus-updates/core/3.0.yaml

```diff
@@ -26,6 +26,13 @@
 #####
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: custom
```

Comment:
@JiriOndrusek is that normal?

Reply:
I forgot to write down a name which makes a sense - this was just my "custom" text, I'll fix it during tomorrow

Reply:
I added the correct text here

Reply:
@JiriOndrusek it's still the old text

Reply:
I believe at some point you have removed a commit or something, make sure that you have all your changes.

Reply:
Thanks good catch @ia3andy I renamed it, is it make okay now ?